runtime/doc/options.txt

```diff
@@ -1386,6 +1386,21 @@ A jump table for the options with a short description can be found at |Q_op|.
 	This option cannot be set from a |modeline| or in the |sandbox|, for
 	security reasons.
 
+						*'completeslash'* *'csl'*
+'completeslash' 'csl'	string	(default: "")
+			local to buffer
+			{not in Vi}  {only for MS-Windows}
```

Comment:
For what it's worth, the rest of the options list "Windows" instead of "MS-Windows". Is there a precedent for fixing these sorts of things when porting patches?

Reply:
Looks like this is already handled in a different patch, see [here](https://github.com/neovim/neovim/pull/13116#discussion_r508112107).